<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="YP1000555DT" failures="0" tests="4" name="com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation" time="12.969" errors="3" timestamp="29 Mar 2018 10:36:39 GMT" skipped="0">
  <testcase classname="com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation" name="Transaction_ProjectAllocationNavigation" time="2.232">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.yash.RMS.POM.Transaction.PageObjects_ProjectAllocation.Link_Transaction(PageObjects_ProjectAllocation.java:19)
at com.yash.RMS.TestAction.Transaction.Action_ProjectAllocation.ProjectAllocationPageNavigation(Action_ProjectAllocation.java:44)
at com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation.Transaction_ProjectAllocationNavigation(Testcase_ProjectAllocation.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.yash.RMS.CustomProperties.XML.XMLRead(XML.java:545)
at com.yash.RMS.CustomProperties.Main.main(Main.java:13)
]]>
    </error>
  </testcase> <!-- Transaction_ProjectAllocationNavigation -->
  <testcase classname="com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation" name="Transaction_AddNewResourceToProject" time="1.836">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.yash.RMS.POM.Transaction.PageObjects_ProjectAllocation.txtbox_ProjectSerach(PageObjects_ProjectAllocation.java:30)
at com.yash.RMS.TestAction.Transaction.Action_ProjectAllocation.AddNewResourceToProject(Action_ProjectAllocation.java:889)
at com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation.Transaction_AddNewResourceToProject(Testcase_ProjectAllocation.java:126)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.yash.RMS.CustomProperties.XML.XMLRead(XML.java:545)
at com.yash.RMS.CustomProperties.Main.main(Main.java:13)
]]>
    </error>
  </testcase> <!-- Transaction_AddNewResourceToProject -->
  <testcase classname="com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation" name="Transaction_EditFResourceDetails" time="7.151"/>
  <testcase classname="com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation" name="Transaction_DeleteResourceFromProject" time="1.750">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.yash.RMS.POM.Transaction.PageObjects_ProjectAllocation.link_ProjectList(PageObjects_ProjectAllocation.java:160)
at com.yash.RMS.TestAction.Transaction.Action_ProjectAllocation.DeleteResourceFromProject(Action_ProjectAllocation.java:1154)
at com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation.Transaction_DeleteResourceFromProject(Testcase_ProjectAllocation.java:166)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at com.yash.RMS.CustomProperties.XML.XMLRead(XML.java:545)
at com.yash.RMS.CustomProperties.Main.main(Main.java:13)
]]>
    </error>
  </testcase> <!-- Transaction_DeleteResourceFromProject -->
</testsuite> <!-- com.yash.RMS.Testcase.Transaction.Testcase_ProjectAllocation -->
